.github/workflows/release.yaml

```diff
@@ -424,3 +424,25 @@ jobs:
             Helm upgrade to and install of `${{ github.ref }}` works.
             ## Checklist
             - [x] PR is labeled accordingly
+      - name: Create pull request for adding helm index to ${{ github.ref }} branch
+        if: ${{ !contains(github.ref, '-rc.') }}
+        uses: peter-evans/create-pull-request@v7
```

Comment:
I know we don't do that consistently, but would be nice to use the sha value here

Reply:
Good idea :+1: done!